py-polars/polars/series/series.py

```diff
@@ -403,6 +403,11 @@ def shape(self) -> tuple[int]:
     @property
     def time_unit(self) -> TimeUnit | None:
         """Get the time unit of underlying Datetime Series as {"ns", "us", "ms"}."""
+        warnings.warn(
+            "In a future version of polars, the `time_unit` attribute will be removed",
```

Comment:
could you advise what to use instead too please?

Reply:
sorry for taking up your time, realise that you may have done it faster than just the review 😅
I'll convert to draft until i fix the tests too - i struggled to set up the environment but i think i'm gonna figure it out now